src/pybind/mgr/dashboard/frontend/src/app/core/navigation/navigation/navigation.component.html

```diff
@@ -257,6 +257,12 @@
                             i18n-title
                             *ngIf="permissions.nfs.read && enabledFeature.nfs"
                             class="tc_submenuitem tc_submenuitem_file_nfs"><span i18n>NFS</span></cds-sidenav-item>
+          <cds-sidenav-item route="/cephfs/smb"
+                            [useRouter]="true"
+                            title="SMB"
+                            i18n-title
+                            *ngIf="permissions.smb.read"
+                            class="tc_submenuitem tc_submenuitem_file_smb"><span i18n>SMB</span></cds-sidenav-item>
```

Comment:
can you update unit-tests and also add basic e2e for this section. you can already find the `navigation.e2e-spec.ts` file where you can the check for the SMB as well. that should be fine for now.